src/it/java/net/spy/memcached/ProtocolBaseCase.java

```diff
@@ -829,7 +829,7 @@ public void xtestGracefulShutdownTooSlow() throws Exception {
   @Test
   public void testStupidlyLargeSetAndSizeOverride() throws Exception {
     Random r = new Random();
-    SerializingTranscoder st = new SerializingTranscoder(Integer.MAX_VALUE);
+    SerializingTranscoder st = new SerializingTranscoder();
```

Comment:
Why make this change?

Reply:
Skip this test in TLS mode for now. Will follow up later